config/rollup-contants.js

```diff
@@ -48,29 +48,34 @@ export const globals = {
     'react-redux': 'reactRedux'
 };
 
-export const rollupConfig = (external, plugins, globals, name, entries = [ globMapper('src/**/index.js') ], outputDir = './') => ([
-    ...[ 'esm', 'cjs' ].map(env => ({
+export const rollupConfig = (external, plugins, globals, name, entries = [ globMapper('src/**/index.js') ], outputDir = './') => {
+    console.warn(JSON.stringify(entries), process.env.FORMAT);
```

Comment:
Do we really want to print out entries and format?

Reply:
no we don't I used it for debug :+1: